wtxmgr/db.go

```diff
@@ -165,6 +169,29 @@ func ReadAddrTxOutput(v []byte, txout *AddrTxOutput) (err error) {
 	return nil
 }
 
+func readAddrTxOutputOld(v []byte, txout *AddrTxOutput) (err error) {
+	defer func() {
+		if rev := recover(); rev != nil {
```

Comment:
when you need to recover from the panic? what kind of panic you want to recover? why?